packages/script/yugabyte

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# yugabyted daemon
+# chkconfig: 345 20 80
+# description: yugabyted daemon
+# processname: yugabyted
+
+DAEMON_PATH="/opt/yugabyte-2.0.5.2"
```

Comment:
ideally, we wouldn't need this separate daemon script because yugabyted supports all the functionality required. it has a start/stop/status command and puts its pid file in the root directory (yugaybted.pid)

Reply:
yugabyted support only start option there is no stop option available in yugabyte version 2.0.5.2. That's why I create a demon script. and when we start it directly, yugabyted create config in root directly, but I think i should create a config file in `/etc/yugabyte` directory, also redirect its logs to `/var/log/yugabyte-logs`
```
vaibhav@instance-3:~$ yugabyted --help
Usage: yugabyted [-h] {start,status,version,demo,connect} ...

To start YugabyteDB, run 'yugabyted start'.

Commands:
  {start,status,version,demo,connect}
    start               Start YugabyteDB.
    status              Print status of YugabyteDB.
    version             Version of YugabyteDB.
    demo                Load and interact with preset demo data.
    connect             Connect to an API through the CLI.

Flags:
  -h, --help            show this help message and exit

Run 'yugabyted [command] -h' for help with specific commands.

```